README.md

```diff
@@ -1,2 +1,68 @@
-# yast-rake-ci
-Extension Rake tasks for the yast-rake gem
+
+YaST CI Tasks
+=============
+
+[![Build Status](http://ci.opensuse.org/buildStatus/icon?job=yast-rake-ci-master)](http://ci.opensuse.org/view/Yast/job/yast-rake-ci-master/)
+[![Code Climate](https://codeclimate.com/github/yast/yast-yast-rake-ci/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-rake-ci)
+
+
+Description
+============
+
+This package provides extensions for the [YaST Rake tasks](https://github.com/yast/yast-rake).
+
+These tasks are needed only during development and in Jenkins build, this package
+in not needed at run-time or in OBS for building the YaST packages.
+
+The package is used to install all needed packages via RPM dependencies.
+
+Provided Tasks
+--------------
+
+### check:ci ###
+
+This Rake task is designed for running tests and check in Jenkins CI. These
+checks can run locally as well. This task is used by this module itself to run
+the tests during package build.
+
+The task runs these tests and checks:
+
+- Runs test using the [`test:unit`](https://github.com/yast/yast-rake#testunit)
+  task, it sets `COVERAGE=1` environment variable to run the tests with the
+  code coverage enabled.
+- If [`.rubocop.yml`](https://github.com/bbatsov/rubocop/blob/master/config/default.yml)
+  file is present it runs [`rubocop`](https://github.com/bbatsov/rubocop)
+  to check the code style
+- If the file is missing it runs simple [`check:syntax`](https://github.com/openSUSE/packaging_tasks#checksyntax)
+  task to at least check the syntax of the files.
+- It runs the [`check:spelling`](https://github.com/yast/yast-rake#checkspelling)
+  task to check spelling in documentation files
```

Comment:
this will create a lot of new failures, so I am not sure if we want to have it from start enabled for all modules